processor/.eslintrc.json

```
@@ -28,6 +28,23 @@
         "variables": true
       }
     ],
+    "@typescript-eslint/array-type": [
+      "error",
+      {
+        "default": "generic"
+      }
+    ],
+    "no-unused-vars": "off",
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      {
+        "vars": "all",
+        "args": "after-used",
+        "ignoreRestSiblings": false,
+        "varsIgnorePattern": "^_",
+        "argsIgnorePattern": "^_"
```

this needs to be `_+` so we can use `const [_, __, ___, used] = ...` etc. i'm not sure the `^` does anything here (really not sure, worth checking). 

```suggestion
        "varsIgnorePattern": "_+",
        "argsIgnorePattern": "_+"
```

---

`^_` will ignore any variable starts with and `_` which means `[_,__,___]` will work perfectly fine. The added benefit here is we can still give them meaningful names: `[used1, _ignored, used2]`.

https://eslint.org/docs/latest/rules/no-unused-vars#argsignorepattern